tests/php/phpstan.sh

```diff
@@ -14,7 +14,7 @@ docker run -tid --rm -v ps-volume:/var/www/html --name temp-ps prestashop/presta
 # Clear previous instance of the module in the PrestaShop volume
 echo "Clear previous module"
 
-docker exec -t temp-ps rm -rf /var/www/html/modules/ps_emailalerts
+docker exec -t --user root temp-ps sh -c 'find /var/www/html/modules/ps_emailalerts -type f -exec rm {} +'
```

Comment:
```suggestion
docker exec -t --user root temp-ps sh -c 'if [ -d "/var/www/html/modules/ps_emailalerts" ]; then find /var/www/html/modules/ps_emailalerts -type f -exec rm {} +; fi'
```

something like this maybe?